src/test/ui/for-loop-while/label_break_value.rs

```diff
@@ -128,8 +128,8 @@ fn label_break_macro() {
         0
     };
     assert_eq!(x, 0);
-    let x: u8 = 'a: { //~ WARNING `'a` shadows a label
-        'b: { //~ WARNING `'b` shadows a label
```

Comment:
Marc file as check-pass

Reply:
It is already `run-pass` because it checks that the correct labels are selected in case of hygienic shadowing.